projects/storefrontapp/src/environments/environment.ts

```diff
@@ -28,12 +28,16 @@ export const environment: Environment = {
   cdc: buildProcess.env.CX_CDC ?? false,
   cdp: buildProcess.env.CX_CDP ?? false,
   cpq: buildProcess.env.CX_CPQ ?? false,
+
```

Comment:
MAJOR: please remove un-intended changes from your PR. follow this for all files in this OR